voltdbclient/distributor.go

```diff
@@ -71,9 +71,28 @@ func newConn(cis []string) (*Conn, error) {
 	return c, nil
 }
 
-// OpenConn returns a new connection to the VoltDB server.  The name is a
-// string in a driver-specific format.  The returned connection can be used by
-// only one goroutine at a time.
+// OpenConn returns a new connection to the VoltDB server.  The name is a string
+// in a driver-specific format.  The returned connection can be used by only one
+// goroutine at a time.
+//
+// By default voltdb doesn't require authentication,
+// clients connecting to un secured database have access to everything.
+// Supplying connection credentials doesn't affect for non secured databases
+//
+// Here we authenticate if username and password are supplied, if they are not
+// then a connection is established without doing the authentication
+//
+// Connection string is similar to postgres, default port is 21212
+//
+// voltdb://
+// voltdb://localhost
+// voltdb://localhost:21212
+// voltdb://user@localhost
+// voltdb://user:secret@localhost
+// voltdb://other@localhost?some_param=some_value
```

Comment:
Yes we do. As you can see from the comment. Username and password can be passed via the url.

It is possible to pass additional values in the form of query params i.e after `?` part of the url but that is beyond the scope of this PR